js/components/NoteList/AddNote/AddNote.js

```diff
@@ -21,28 +25,14 @@ export default class AddNote extends Component {
           onRequestClose={() => close(false)}
         >
           <View style={styles.container}>
-            <View style={styles.modal}>
-              <View style={styles.header}>
-                <Text style={styles.title}>Add Note:</Text>
-                <TouchableOpacity
-                  activeOpacity={0.5}
-                  onPress={() => close(false)}
-                >
-                  <Text style={styles.close}>X</Text>
-                </TouchableOpacity>
-              </View>
-              <TextInput
-                {...this.props}
-                multiline={true}
-                onChangeText={text => this.setState({ text })}
-                autoFocus
-                value={this.state.text}
-                style={styles.textInput}
-              />
-              <TouchableOpacity activeOpacity={0.5} style={styles.button}>
-                <Text style={styles.submit}>Submit</Text>
-              </TouchableOpacity>
-            </View>
+            <MessageBox
+              close={close}
+              updateText={text => this._updateText(text)}
+              showCloseIcon
+              value={this.state.text}
```

Comment:
the final form takes care of this. so you do not need to pass this as a prop and have a state for that.